lib/cli.rb

```diff
@@ -920,10 +920,6 @@ def self.parse_inspect_command_options(host, options)
 
       if options[:public]
```

Comment:
Here we map the `:public` option to an ip address and in `ServeHtmlTask` it is done the other way around. This could be simplified by directly passing on the `:public` option.